python/dgl/transform.py

```diff
@@ -1886,12 +1888,19 @@ def to_simple(g,
     copy_edata: bool, optional
         If True, the edge features of the simple graph are copied
         from the original graph. If there exists duplicate edges between
-        two nodes (u, v), the feature of the edge is randomly selected
-        from one of the duplicate edges.
+        two nodes (u, v), the feature of the edge is the aggregation
+        of edge feature of duplicate edges.
 
         If False, the simple graph will not have any edge features.
 
         (Default: False)
+    aggregator: str, optional
+        Indicate how to aggregate edge feature of duplicate edges.
+        If ``random``, randomly select one of the duplicate edges' feature.
```

Comment:
I realized that it's not really "random".  I would use "arbitrary" instead ("Arbitrary" means "at one's discretion").